modules/preview-react/table/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './lib/Table';
+export * from './lib/TableCaption';
+export * from './lib/TableDivider';
+export * from './lib/TableBody';
+export * from './lib/TableCell';
+export * from './lib/TableFooter';
+export * from './lib/TableHeader';
+export * from './lib/TableHead';
+export * from './lib/TableRow';
```

Comment:
Is there anything else exported that might be useful on the files that aren't Table? Table itself attaches all the other sub components and I'm not sure if you can use `TableFooter` without the parent `Table` component. Look at Modal's index.ts file for example or Pills index.ts

Reply:
If you tried to use `TableRow` without table it would give you an error right?

Reply:
Yeah definitely good point. I will update this because it will give an error